### PR TITLE
Added cw-plus interface

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims at adding cw-plus interface to the cw-orch repo.
The artifacts for cw-plus are not located directly in the github repo but need to be dowloaded before publishing the cw-plus-orch package to crates.io to make sure that the wasms are available !

### Checklist

- [ ] Changelog updated.
- [ ] Docs updated.

### CODE REVIEW

File: before_publish.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+cargo run --example download-wasms -p cw-plus-orch
```
Comment (Buckram123) [inline before_publish.sh:1]:
```suggestion
cargo run --example download-wasms -p cw-plus-orch

```

File: packages/integrations/cw-plus/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+[package]
+name = "cw-plus-orch"
+version = "0.25.0"                                                 # Version of cw-orch that's used
+edition = "2021"
+description = "CW4 implementation of group based on staked tokens"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/cw-plus"
+homepage = "https://cosmwasm.com"
+documentation = "https://docs.cosmwasm.com"
```
Comment (Buckram123) [inline packages/integrations/cw-plus/Cargo.toml:9]:
```suggestion
description = "cw-orch interfaces for cw-plus base contracts"
license = "Apache-2.0"
repository = "https://github.com/AbstractSDK/cw-orchestrator"
homepage = "https://abstract.money"
documentation = "https://orchestrator.abstract.money"
```

File: packages/integrations/cw-plus/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+
```
Comment (Buckram123) [inline packages/integrations/cw-plus/Cargo.toml:14]:
```suggestion
```

No features right now

File: packages/integrations/cw-plus/examples/download-wasms.rs
Comment (Buckram123) [inline packages/integrations/cw-plus/examples/download-wasms.rs:1]: Rename to snake_case

File: packages/integrations/cw-plus/examples/test-wasm.rs
Comment (Buckram123) [inline packages/integrations/cw-plus/examples/test-wasm.rs:1]: Rename to snake_case

File: .gitignore [hunk 4]
```diff
@@ -21,4 +21,6 @@ gen_cov_test
 
 **/logs/
 
-docs/book
\ No newline at end of file
+docs/book
+
+packages/integrations/**/artifacts
```
Comment (Buckram123) [inline .gitignore:26]:
```suggestion
packages/integrations/**/artifacts

```

**Review by Buckram123 — APPROVED** [review]
Thank you for moving it to cw-orch!

File: packages/integrations/cw-plus/examples/download-wasms.rs
Comment (Buckram123) [reply packages/integrations/cw-plus/examples/download-wasms.rs:1]: (file)

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**